ansible/roles-infra/infra-osp-template-create/defaults/main.yml

```diff
@@ -10,3 +10,6 @@ cf_tags:
 cloud_tags: {}
 
 cloudformation_retries: 0
+
+max_update_in_progress: 10
+max_create_in_progress: 10
```

Comment:
New variables should have prefixes that match the role, cloud provider, or config.

I think cloud provider prefix is appropriate here and would recommend variable naming `osp_max_update_in_progress` and `osp_max_create_in_progress`.

Reply:
Fixed in 93ab338